src/components/asset-list/RecyclerAssetList2/WrappedNFT.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useCallback } from 'react';
+import { UniqueTokenCard } from '../../unique-token';
+import { Box, BoxProps } from '@rainbow-me/design-system';
+import { useAsset, useWallets } from '@rainbow-me/hooks';
+import { useNavigation } from '@rainbow-me/navigation';
+import Routes from '@rainbow-me/routes';
+
+export default function WrappedNFT({
+  uniqueId,
+  placement,
+}: {
+  uniqueId: string;
+  placement: 'left' | 'right';
+}) {
+  const asset = useAsset({ type: 'nft', uniqueId });
+  const { isReadOnlyWallet } = useWallets();
```

Comment:
since this is already pulled into the `UniqueTokenExpandedState` we don't need to pass it in here